src/components/UserPage.js

```diff
@@ -1,14 +1,17 @@
 import React, { useState, useEffect } from "react";
-import { Link, useParams } from "react-router-dom";
+import { connect, useSelector } from "react-redux";
+import { Link, useParams, withRouter } from "react-router-dom";
 import { Container } from "reactstrap";
 import axios from "axios";
+import { getUserVideos } from "../store/actions";
 
-const UserPage = () => {
+const UserPage = props => {
   const { username } = useParams();
   const [videos, setVideos] = useState([]);
 
   useEffect(() => {
-    axios
+    props.getUserVideos(localStorage.getItem("token"), username);
+    /*axios
       .get(`http://localhost:5050/api/users/username/${username}`)
```

Comment:
We'll want to remove the commented out stuff

Reply:
completed